client/web-sveltekit/src/lib/repo/filePopover/FilePopover.svelte

```diff
@@ -0,0 +1,269 @@
+                align-items: center;
+                border-bottom: 1px solid var(--border-color);
+                display: flex;
+                flex-flow: row nowrap;
+                gap: 0.25rem;
+                justify-content: flex-start;
+                padding: 0.5rem 1rem;
+                width: 100%;
+                font-family: var(--monospace-font-family);
+                white-space: nowrap;
+                color: var(--text-muted);
+
+                small {
+                    overflow: hidden;
+                    text-overflow: ellipsis;
+                    white-space: nowrap;
+                }
+            }
+
+            .lang-and-file {
+                width: 100%;
+                display: flex;
+                flex-flow: row nowrap;
+                align-items: center;
+                justify-content: flex-start;
+                padding: 0.5rem 1rem;
+                gap: 0.25rem 0.75rem;
+
+                .file {
+                    display: flex;
+                    flex-flow: column nowrap;
+                    align-items: flex-start;
+                    justify-content: flex-start;
+                    font-family: var(--monospace-font-family);
+                    gap: 0.25rem;
+
+                    div {
+                        color: var(--text-body);
+                    }
+
+                    small {
+                        color: var(--text-muted);
+                    }
+                }
+            }
+        }
+
+        .last-commit {
+            display: flex;
+            flex-flow: column nowrap;
+            align-items: flex-start;
+            justify-content: center;
+            gap: 0.5rem 0.5rem 0rem;
+
+            .title {
+                padding: 0.5rem 1rem;
+                color: var(--text-body);
+                background-color: var(--secondary-4);
+                width: 100%;
+                border-bottom: 1px solid var(--border-color);
+            }
+
+            .commit {
+                padding-left: 1.5rem;
+                display: flex;
+                flex-flow: row nowrap;
+                align-items: center;
+                justify-content: flex-start;
+                width: 100%;
+                height: 90px;
+                gap: 0.5rem 1.25rem;
+
+                div {
+                    display: flex;
+                    flex-flow: column nowrap;
+                    align-items: flex-start;
+                    justify-content: center;
+                    gap: 0.25rem;
+                    width: 325px;
+
+                    a {
+                        font-family: var(--monospace-font-family);
+                        background-color: var(--color-bg-2);
+                        padding: 0.15rem 0.25rem;
+                        border-radius: 3px;
+                    }
+
+                    .msg {
+                        color: var(--text-body);
+                        text-overflow: ellipsis ellipsis;
```

Comment:
Only one `ellipsis` is needed. The two-param form is for when you want different behavior for clipping the start and end of the text.

```suggestion
                        text-overflow: ellipsis;
```

Reply:
Also, it looks like this might not be working? Maybe because the width is set explicitly